spec/views/companies/_map_companies.haml_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+class TempCompanyHelper
+  include CompaniesHelper
+end
+
+RSpec.describe "company maps partial: _map_companies.html.haml" do
+
+  it "map markers are raw json (not escaped)" do
+
+    test_companies = []
+    test_companies << create(:company)
```

Comment:
You could collapse these two statements into:
```
test_companies = [ create(:company) ]
```
but that is simply a matter of preference.